doc/sphinx/language/core/definitions.rst

```diff
@@ -203,12 +201,12 @@ commands to manage the proof mode (see :ref:`proofhandling`).
 
 When the proof is complete, use the :cmd:`Qed` command so the kernel verifies
 the proof and adds it to the global environment. By default, proofs
-ended by :cmd:`Qed` are sealed, that is that their content cannot
+that end with :cmd:`Qed` are sealed, that is that their content cannot
 be unfolded (see :ref:`applyingconversionrules`), thus realizing some
-form of *proof-irrelevance*. Proofs can be made unfoldable, like
-definitions are, by using the attribute :attr:`defined` or by ending
-the proof with :cmd:`Defined` in place of :cmd:`Qed`. Using the
-attribute is however recommended.
+form of *proof-irrelevance*. Proofs can be made unfoldable, as
```

Comment:
Think you missed my earlier comment:

> "some form of proof-irrelevance"?? Rather vague, can you be more specific?

Reply:
I now wrote "... thus realizing *proof irrelevance*, that is that only provability matters, and not the exact proof". Is that better?